src/Angor/Client/Pages/Create.razor

```diff
@@ -204,352 +189,598 @@
+    {
+        ProjectInfo = new ProjectInfo
         {
-            ProjectInfo = new ProjectInfo
-            {
-                StartDate = DateTime.UtcNow.AddHours(2),
-                PenaltyDays = 90,
-                ExpiryDate = DateTime.UtcNow.AddDays(120),
-                TargetAmount = 50,
-            },
-            Metadata = new(),
-            CreationTransactionId = null,
-        };
+            StartDate = DateTime.UtcNow.AddHours(2),
+            PenaltyDays = 90,
+            ExpiryDate = DateTime.UtcNow.AddDays(120),
+            TargetAmount = 50
+        },
+        Metadata = new ProjectMetadata(),
+        CreationTransactionId = null
+    };
+    
+    private readonly Dictionary<string, (int[] Percentages, int IntervalDays)> presets = new()
+    {
+        { "preset1", (new[] { 10, 30, 60 }, 10) },
+        { "preset2", (new[] { 25, 25, 25, 25 }, 10) },
+        { "preset3", (new[] { 60, 20, 20 }, 15) }
+    };
+    
 
     private int activeTab = 1;
+    
+    private int totalDuration;
+    private int numberOfStages;
 
-    bool createProfileSpinner = false;
-    bool createApplicationDataSpinner = false;
-    bool creatProjectSpinner = false;
-    bool publishProjectSpinner = false;
+    bool createProfileSpinner;
+    bool createApplicationDataSpinner;
+    bool creatProjectSpinner;
+    bool publishProjectSpinner;
 
-    bool nostrMetadataCreated = false;
-    bool nostrApplicationSpecificDataCreated = false;
+    bool nostrMetadataCreated;
+    bool nostrApplicationSpecificDataCreated;
 
     protected override async Task OnInitializedAsync()
     {
-        if (hasWallet)
+        if (!hasWallet)
         {
-            var projects = storage.GetFounderProjects();
-            var keys = _walletStorage.GetFounderKeys();
+            return;
+        }
 
-            if (projects.Where(_ => !string.IsNullOrEmpty(_.CreationTransactionId)).ToList().Count == keys.Keys.Count)
-            {
-                notificationComponent.ShowErrorMessage("All founder keys have been used for this wallet!");
-                return;
-            }
+        var projects = storage.GetFounderProjects() ?? new List<FounderProject>();
+        var keys = _walletStorage.GetFounderKeys();
 
-            var latestProject = projects.Where(_ => !string.IsNullOrEmpty(_.CreationTransactionId)).MaxBy(p => p.ProjectIndex);
-            var projectsKeys = _derivationOperations.GetProjectKey(keys, latestProject?.ProjectIndex + 1 ?? 1);
+        if (projects.Count(p => !string.IsNullOrEmpty(p.CreationTransactionId)) >= keys.Keys.Count)
+        {
+            notificationComponent.ShowErrorMessage("All founder keys have been used for this wallet!");
+            return;
+        }
 
-            var fromStorage = projects.FirstOrDefault(p => p.ProjectInfo.ProjectIdentifier == projectsKeys.ProjectIdentifier);
+        var latestProject = projects
+            .Where(p => !string.IsNullOrEmpty(p.CreationTransactionId))
```

Comment:
I in this new approach we do not push the project information to storage until we call the method created profile button.
It means you cannot navigate away and come back to continue editing the project.

Perhaps we should push to storage the new project immediately now and later update it as we create metadata
For example when we navigate away from each tab we could save the project to storage.
Until the create profile was called then we do not modify anymore the project class (we must also make sure we save to storage the fact the profile was created (not sure how we do it right now)